scrapingbee_fetch_news/program.py

```diff
@@ -6,15 +6,19 @@
 from autokitteh.slack import slack_client
 
 SB_API_KEY = os.getenv("SB_API_KEY")
-NEWS_URL = os.getenv("NEWS_URL", "https://www.theguardian.com/international/rss")
+NEWS_WEBSITE_URL = os.getenv(
```

Comment:
I think you should put the "demo" URL in the `autokitteh.yaml` and not in the python, because the user might not understand where from the URL comes from, since the ENV Var is empty.
And here in python, you should check if `NEWS_WEBSITE_URL` doesn't exist in env-vars : raise an error.

In my eyes, it's better to include the "demo" URL in the `autokitteh.yaml` file rather than in the Python code, as users may not comprehend the source of the URL when the ENV variable is empty (for example - they see that the ENV variable is empty in the WebUI - but they still get a URL) . Additionally, in the Python code, it is essential to verify whether `NEWS_WEBSITE_URL` is absent from the environment variables; if so, an error should be raised.